packages/react/src/portal.tsx

```diff
@@ -48,6 +50,10 @@ export const Portal: (props: PortalProperties & RefAttributes<ComponentInternals
           const [width, height] = size.value
           const dpr = store.getState().viewport.dpr
           renderTarget.setSize(width * dpr, height * dpr)
+          injectState.camera!.aspect = width / height;
```

Comment:
As commented before, this should be done using the store.getState().camera (not literally and not here because you cannot get the store from inside the portal here)
Therefore, imo the best approach is to get the store `useStore` inside `ChildrenToFBO` and subscribe to it inside a useEffect and apply the size of the state form the store to the camera of the state from the state

Reply:
I see your point, but initally we need to set the position and rotation of the portal camera to something non-zero. The r3f Portals example sets it to the scenere cameras position and rotation, which we only have access here since injectState will override it until we get to useFrame. So I think we need both. Here, we should set the position and rotation of the camera. Then in useFrame, we can update the aspect ratio based on the size.

Reply:
I was not referring the the position code. That part is fine although we could think about directly setting the position when creating the camera inside the memo. I am also not sure copying the camera's transformation is better then just setting it to something non zero. Copying could lead to unexpected behavior, e.g. "in this example it works" but here it soundenly doesn't work"